packages/api-reference/src/components/Section/SectionAccordion.vue

```diff
@@ -66,33 +66,28 @@ const isHovered = useElementHover(button)
   z-index: 0;
 }
 /* Use a pseudo elements so we can use CSS filters to lighten the background  */
-.section-accordion::before,
-.section-accordion::after {
+.section-accordion::before {
```

Comment:
If we're not doing the color opacity anymore this doesn't need to be a pseudo element, we can just use a `background`